src/lsp/cobol_parser/grammar.mly

```diff
@@ -3453,9 +3453,12 @@ let generate_statement :=
 let go_to_statement :=
   | GO; TO?; %prec lowest
     { LoneGoTo }    (* COB85; obsolete; should be sole statement of paragraph *)
-  | GO; TO?; targets = nel_(procedure_name);
-    depending_on = o(DEPENDING; ON?; ident);
-    { GoTo (GoToSimple { targets; depending_on }) }
+  | GO; TO?; target = procedure_name;
+    depending_on_clause = o(pair(rl(procedure_name), DEPENDING; ON?; ident));
```

Comment:
If using `ro` instead of `o` here still passes the tests, I think that would be preferable (since that's more recovery-friendly).